package-lock.json

```diff
@@ -5029,6 +5029,18 @@
         "url": "https://github.com/sponsors/ljharb"
       }
     },
+    "node_modules/array.prototype.tosorted": {
```

Comment:
Always try to include any dependencies you add in your PR description. 

What did you add? Why did you add it? Stuff like that.

Reply:
Oh I think I did add one. This was also to try and fix my silly error. It should be this one: https://www.npmjs.com/package/eslint-plugin-react